api/src/services/healthService.js

```diff
@@ -8,26 +8,21 @@ const SEPARATOR = ',';
 
 async function checkAccountsBalances(req, res) {
   const { query } = req;
-  let accounts;
-  let blockchains;
-  let networks;
-  let groups;
-  let addresses;  
-
-  try {
-    blockchains = splitParams(query.blockchains);
-    validateBlockchains(blockchains);
-    networks = splitParams(query.networks);
-    validateNetworksNames(networks);
-    groups = splitParams(query.groups);
-    validateGroups(groups);
-    accounts = cacheService.getAccounts();
-    addresses = splitParams(query.addresses);
-    validateAddresses(addresses, accounts);
-  } catch (error) {
-    return res.status(400).json({
-      msg: error.message,
-    });
+  const blockchains = splitParams(query.blockchains);
+  const networks = splitParams(query.networks);
+  const groups = splitParams(query.groups);
+  const addresses = splitParams(query.addresses);
+  const accounts = cacheService.getAccounts();
+  
+  let validationErrors = [
+    ...validateBlockchains(blockchains),
+    ...validateNetworks(networks),
+    ...validateGroups(groups),
+    ...validateAddresses(addresses, accounts)
+  ];
+
+  if (validationErrors.length) {
+    return res.status(400).json({ msg: validationErrors });
```

Comment:
`errors` instead of `msg`

Reply:
msg -> "Validation failed"
errors -> list of errors

Reply:
fixed